tests/format/json/json/__snapshots__/format.test.js.snap

```diff
@@ -594,15 +594,16 @@ No \\\\n's!",
   },
   {
     "//": "JSON5 numbers",
-    "hexadecimal": 912559,
-    "leadingDecimalPoint": 0.8675309,
-    "andTrailing": 8675309,
+    "hexadecimal": 0xdecaf,
+    "leadingDecimalPoint": .8675309,
+    "andTrailing": 8675309.,
```

Comment:
Not fully sure we want to stop formatting even in these two cases. Prettier removes trailing commas and useless fractional digits in CSS, for instance. This is good:

```diff
-font-size: 13.0px;
-line-height: 16.20000px;
+font-size: 13px;
+line-height: 16.2px;
```

Should we still format some (but not all) values in JSON?